Engine/source/Sound/CSoundAnalysisMP3.cpp

```diff
@@ -225,7 +225,7 @@ bool SoundAnalysis_MP3( const char* _path, sSoundAnalysisData* _analysisData )
         
         // フレームヘッダ読み込み
 		decryptor.gotoOffset(skipOffset);
-        fseek( fp, skipOffset + (decryptor.m_useNew ? 4 : 0), SEEK_SET );
+        fseek( fp, skipOffset + (decryptor.m_useNew ? 16 : 0), SEEK_SET );
```

Comment:
it's better to use ```decryptor.m_header_size``` instead of ```(decryptor.m_useNew ? 16 : 0)```, because it will fail if file encrypted with 2 version